paparazzi/paparazzi-gradle-plugin/src/test/projects/complex-a11y/src/test/java/app/cash/paparazzi/plugin/test/ComplexA11yTest.kt

```diff
@@ -0,0 +1,21 @@
+package app.cash.paparazzi.plugin.test
+
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.Paparazzi
+import app.cash.paparazzi.accessibility.AccessibilityRenderExtension
+import org.junit.Rule
+import org.junit.Test
+
+class ComplexA11yTest {
```

Comment:
nit: can we rename ComplexA11y and complex-a11y everywhere to compose-a11y?  It's only complex in implementation detail to us, but should be easy from the user perspective ;)